app/Http/Controllers/Wiki/LanguagesController.php

```diff
@@ -212,10 +211,11 @@ public function search($search_text)
                 $media,
                 $set_type_code
             ]
-        );
-        $languages = cacheRemember(
-            'languages_search',
-            $cache_params,
+        ;
+        $cache_key = generateCacheSafeKey('languages_search', $cache_params);
```

Comment:
We will create a safe key before we call the cacheRememberByKey method. For now, it will be implemented for the searching endpoints